### PR TITLE
AC-561: Add a privacy policy

### PR DESCRIPTION
**Pull request by deepak140596**
## Description of what I changed

Added a privacy policy in Markdown format at the root of the repository

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-561

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: privacy_policy.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+**Changes to This Privacy Policy**
+
+We may update our Privacy Policy from time to time. Thus, you are advised to review this page periodically for any changes. We will notify you of any changes by posting the new Privacy Policy on this page. These changes are effective immediately after they are posted on this page.
+
+**Contact Us**
+
+If you have any questions or suggestions about our Privacy Policy, do not hesitate to contact us.
+
+This privacy policy page was created at [privacypolicytemplate.net](https://privacypolicytemplate.net) and modified/generated by [App Privacy Policy Generator](https://app-privacy-policy-generator.firebaseapp.com/)
```
Comment (VANKINEENITAWRUN) [inline privacy_policy.md]: @deepak140596 I think we can remove the last 2 lines regarding the privacy policy creation.

File: privacy_policy.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+## Privacy Policy
+
+OpenMRS built the OpenMRS Client app as an Open Source app. This SERVICE is provided by OpenMRS at no cost and is intended for use as is.
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
OpenMRS built the OpenMRS Android client app as an Open Source app. This service is provided by OpenMRS at no cost and is intended for use as is.
```

File: privacy_policy.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+This page is used to inform visitors regarding our policies with the collection, use, and disclosure of Personal Information if anyone decided to use our Service.
```
Comment (f4ww4z) [inline privacy_policy.md]: Watch capital letters.
```suggestion
This page is used to inform visitors regarding our policies with the collection, use, and disclosure of personal information if anyone decided to use our service.
```

File: privacy_policy.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+If you choose to use our Service, then you agree to the collection and use of information in relation to this policy. The Personal Information that we collect is used for providing and improving the Service. We will not use or share your information with anyone except as described in this Privacy Policy.
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
If you choose to use our service, then you agree to the collection and use of information in relation to this policy. The personal information that we collect is used for providing and improving the service. We will not use or share your information with anyone except as described in this Privacy Policy.
```

File: privacy_policy.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+The terms used in this Privacy Policy have the same meanings as in our Terms and Conditions, which is accessible at OpenMRS Client unless otherwise defined in this Privacy Policy.
```
Comment (f4ww4z) [inline privacy_policy.md]: @deepak140596 Do we have a terms and conditions page? If not, I think you should rephrase this and mention https://openmrs.org/license/ .

File: privacy_policy.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+**Information Collection and Use**
```
Comment (f4ww4z) [inline privacy_policy.md]: add a new line. Make this into a header instead.

File: privacy_policy.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+For a better experience, while using our Service, we may require you to provide us with certain *personally identifiable information*, including but not limited to
```
Comment (f4ww4z) [inline privacy_policy.md]: remove unnecessary capital letters. Remove 'certain'

File: privacy_policy.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+The terms used in this Privacy Policy have the same meanings as in our Terms and Conditions, which is accessible at OpenMRS Client unless otherwise defined in this Privacy Policy.
+
+**Information Collection and Use**
+For a better experience, while using our Service, we may require you to provide us with certain *personally identifiable information*, including but not limited to 
+
+1. Patient's full name
+2. Patient’s full address
+3. Patient’s Gender, Birth-date and Age
+4. Patient’s Photographs
+5. Patient's Provider Information
```
Comment (f4ww4z) [inline privacy_policy.md]: You can remove `Patient's` from these 5 points. Remove unnecessary capital letters.

File: privacy_policy.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+6. Notes on symptoms and other information
+7. Medical Diagnostics and Vitals (Body temperature, pulse rate, blood pressure,etc)
```
Comment (f4ww4z) [inline privacy_policy.md]: List all of them.

File: privacy_policy.md [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+8. Medical treatment information
+9. Medical appointments
+
+The information that we request will be retained by us and used as described in this privacy policy.
+The app does use third party services that may collect information used to identify you.
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
The app does use third party services that may collect information used to identify you, except [Google Play Services](https://www.google.com/policies/privacy/).
```
Remove the 2 lines below.

File: privacy_policy.md [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+Link to privacy policy of third party service providers used by the app
+
+*   [Google Play Services](https://www.google.com/policies/privacy/)
+
+**Log and App Usage Data**
+1.  App’s log data – This log data may include certain outputs of the app, success and failure states of network calls, etc.
+2.  App usage data - This App Usage Data may include information such as your computer’s IP address, device model and version, device identifier, operating system, time zone, name of your mobile operator or ISP, application performance measurements, and error conditions
```
Comment (f4ww4z) [inline privacy_policy.md]: @deepak140596 Are you sure the Android client is collecting all of these info? If not, just remove this line.

File: privacy_policy.md [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+
+**Cookies**
+Cookies are files with a small amount of data that are commonly used as anonymous unique identifiers. These are sent to your browser from the websites that you visit and are stored on your device's internal memory.
+
+This Service does not use these “cookies” explicitly. However, the app may use third party code and libraries that use “cookies” to collect information and improve their services. You have the option to either accept or refuse these cookies and know when a cookie is being sent to your device. If you choose to refuse our cookies, you may not be able to use some portions of this Service.
```
Comment (f4ww4z) [inline privacy_policy.md]: Watch out for unnecessary capital letters.

File: privacy_policy.md [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+
+**Service Providers**
```
Comment (f4ww4z) [inline privacy_policy.md]: This is not a PWA app. Please remove this section.

File: privacy_policy.md [hunk 14]
```diff
@@ -0,0 +1,95 @@
...
+
+*   To facilitate our Service;
+*   To provide the Service on our behalf;
+*   To perform Service-related services; or
+*   To assist us in analyzing how our Service is used.
+
+We want to inform users of this Service that these third parties have access to your Personal Information. The reason is to perform the tasks assigned to them on our behalf. However, they are obligated not to disclose or use the information for any other purpose.
+
+**Data Storage**
+The app is meant for both Offline as well as Online use. Therefore, we store the data both locally on the device as well as on our servers.
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
Our app may be used when offline as well as online. Therefore, we store the data both locally on the device as well as in our servers.
```

File: privacy_policy.md [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+
+**Security**
```
Comment (f4ww4z) [inline privacy_policy.md]: Make this into a header.

File: privacy_policy.md [hunk 16]
```diff
@@ -0,0 +1,95 @@
...
+We may employ third-party companies and individuals due to the following reasons:
+
+*   To facilitate our Service;
+*   To provide the Service on our behalf;
+*   To perform Service-related services; or
+*   To assist us in analyzing how our Service is used.
+
+We want to inform users of this Service that these third parties have access to your Personal Information. The reason is to perform the tasks assigned to them on our behalf. However, they are obligated not to disclose or use the information for any other purpose.
+
+**Data Storage**
```
Comment (f4ww4z) [inline privacy_policy.md]: Header

File: privacy_policy.md [hunk 17]
```diff
@@ -0,0 +1,95 @@
...
+The app is meant for both Offline as well as Online use. Therefore, we store the data both locally on the device as well as on our servers.
+
+**Security**
+We value your trust in providing us your Personal Information, thus we are striving to use commercially acceptable means of protecting it. But remember that no method of transmission over the internet, or method of electronic storage is 100% secure and reliable, and we cannot guarantee its absolute security. Some of the security measures taken by us are:
+
+*Client Side*
```
Comment (f4ww4z) [inline privacy_policy.md]: Make this into a smaller header.

File: privacy_policy.md [hunk 18]
```diff
@@ -0,0 +1,95 @@
...
+1. Hashing Users’ Password – this App uses [BCrypt](https://en.wikipedia.org/wiki/Bcrypt) hashing function.
+2.  Local Database Encryption – Uses [AES-256 Cipher](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) (using a combination of username and password as its key)
+
+*Server Side*
```
Comment (f4ww4z) [inline privacy_policy.md]: Small header.

File: privacy_policy.md [hunk 19]
```diff
@@ -0,0 +1,95 @@
...
+We want to inform users of this Service that these third parties have access to your Personal Information. The reason is to perform the tasks assigned to them on our behalf. However, they are obligated not to disclose or use the information for any other purpose.
+
+**Data Storage**
+The app is meant for both Offline as well as Online use. Therefore, we store the data both locally on the device as well as on our servers.
+
+**Security**
+We value your trust in providing us your Personal Information, thus we are striving to use commercially acceptable means of protecting it. But remember that no method of transmission over the internet, or method of electronic storage is 100% secure and reliable, and we cannot guarantee its absolute security. Some of the security measures taken by us are:
+
+*Client Side*
+1. Hashing Users’ Password – this App uses [BCrypt](https://en.wikipedia.org/wiki/Bcrypt) hashing function.
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
1. Password hashing: [BCrypt](https://en.wikipedia.org/wiki/Bcrypt) hashing function.
```

File: privacy_policy.md [hunk 20]
```diff
@@ -0,0 +1,95 @@
...
+2.  Local Database Encryption – Uses [AES-256 Cipher](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) (using a combination of username and password as its key)
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
2.  Local database encryption – [AES-256 Cipher](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) (using a combination of username and password as its key)
```

File: privacy_policy.md [hunk 21]
```diff
@@ -0,0 +1,95 @@
...
+
+*Server Side*
+1.  Patient’s Identifiers are Validated using [Mod10](https://en.wikipedia.org/wiki/Luhn_algorithm) algorithm. This is used while creating new Patient’s Identifiers and avoid errors.
+2.  Data is transmitted through a Secure SSL Connection
+3.  Strings are Hashed using [SHA-512](https://en.wikipedia.org/wiki/SHA-2)
+
+**Data Retention**
```
Comment (f4ww4z) [inline privacy_policy.md]: Header.

File: privacy_policy.md [hunk 22]
```diff
@@ -0,0 +1,95 @@
...
+We retain all the data generated by the app and web client. If an account is deactivated/deleted, we move the data related to that account away from our active database BUT is retained in our Backup database.
+
+**How do we use your data**
```
Comment (f4ww4z) [inline privacy_policy.md]: Turn into a header. Change to 'Further notes'

File: privacy_policy.md [hunk 23]
```diff
@@ -0,0 +1,95 @@
...
+1. Hashing Users’ Password – this App uses [BCrypt](https://en.wikipedia.org/wiki/Bcrypt) hashing function.
+2.  Local Database Encryption – Uses [AES-256 Cipher](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) (using a combination of username and password as its key)
+
+*Server Side*
+1.  Patient’s Identifiers are Validated using [Mod10](https://en.wikipedia.org/wiki/Luhn_algorithm) algorithm. This is used while creating new Patient’s Identifiers and avoid errors.
+2.  Data is transmitted through a Secure SSL Connection
+3.  Strings are Hashed using [SHA-512](https://en.wikipedia.org/wiki/SHA-2)
+
+**Data Retention**
+We retain all the data generated by the app and web client. If an account is deactivated/deleted, we move the data related to that account away from our active database BUT is retained in our Backup database.
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
We retain all the data generated by the Android app and web client. If an account is deactivated/deleted, we move the data related to that account away from our active database **but** is retained in our backup database.
```

File: privacy_policy.md [hunk 24]
```diff
@@ -0,0 +1,95 @@
...
+
+**How do we use your data**
+1.  We do not sell your data
+2.  All these personal details helps in creating a record of a certain patient. It helps the Users of this app (the Providers) to maintain the track of the patient.
```
Comment (f4ww4z) [inline privacy_policy.md]: @deepak140596 Watch for unnecessary capital letters.

File: privacy_policy.md [hunk 25]
```diff
@@ -0,0 +1,95 @@
...
+3.  For users’ assistance we may require the personal details of the Patients (such as DoB, name, etc) to easily track the patient and find their records
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
3.  For users’ assistance we may require the personal details of the Patients (such as DoB, name, etc) to easily track the patient and find their records.
```

File: privacy_policy.md [hunk 26]
```diff
@@ -0,0 +1,95 @@
...
+4.  We may aggregate and combine all personal information that we receive through the app from the Patient’s section as well as from the app’s log and usage data and use them to provide better experience across our platforms (both Web and Mobile)
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
4.  We may aggregate and combine all personal information that we receive through this app (see the _Information Collection and Use_) as well as from the app’s log and usage data and use them to provide better experience across our platforms (both Web and mobile).
```

File: privacy_policy.md (hunk 1 above)
Comment (f4ww4z) [reply privacy_policy.md]: Agreed.

File: privacy_policy.md (hunk 11 above)
Comment (deepak140596) [reply privacy_policy.md]: I am not sure about the App usage data but time-zone, error conditions are being collected. 
For log's data, I am sure that that all the log outputs are being collected when a user tries to send a crash report.

File: privacy_policy.md (hunk 5 above)
Comment (deepak140596) [reply privacy_policy.md]: I couldn't find the terms and conditions page, so I rephrased it as

The terms used in this Privacy Policy are subjected to [OpenMRS License](https://openmrs.org/license/).

File: privacy_policy.md (hunk 11 above)
Comment (deepak140596) [reply privacy_policy.md]: @f4ww4z , I have updated the policy, except for the "app usage data". I am still not sure whether to keep it or remove it.

File: privacy_policy.md [hunk 27]
```diff
@@ -0,0 +1,84 @@
...
+The terms used in this Privacy Policy are subjected to [OpenMRS License](https://openmrs.org/license/).
+
+
+## Information Collection and Use
+For a better experience, while using our service, we may require you to provide us with *personally identifiable information*, including but not limited to 
+
+1. Full name
+2. Full address
+3. Gender, birth-date and age
+4. hotographs
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
4. Your photographs
```

File: privacy_policy.md [hunk 28]
```diff
@@ -0,0 +1,84 @@
...
+5. Provider information
+6. Notes on symptoms and other information
+7. Medical diagnostics and vitals (Body temperature, pulse rate, blood pressure,weight, height,respiratory rate)
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
7. Medical diagnostics and vitals (body temperature, pulse rate, blood pressure, weight, height, and respiratory rate)
```

File: privacy_policy.md (hunk 11 above)
Comment (f4ww4z) [reply privacy_policy.md]: You should remove everything except time zone and error conditions. And keep it short.
```suggestion
2.  App usage data: your time zone, application performance measurements, and error conditions.
```

File: privacy_policy.md [hunk 29]
```diff
@@ -0,0 +1,84 @@
...
+9. Medical appointments
+
+The information that we request will be retained by us and used as described in this privacy policy.
+
+Link to privacy policy of third party service providers used by the app
+
+*   [Google Play Services](https://www.google.com/policies/privacy/)
+
+### Log and App Usage Data
+1.  App’s log data – This log data may include certain outputs of the app, success and failure states of network calls, etc.
```
Comment (f4ww4z) [inline privacy_policy.md]:
```suggestion
1.  App’s log data: such as certain outputs of the app, success and failure states of network calls, etc.
```

File: privacy_policy.md [hunk 30]
```diff
@@ -0,0 +1,84 @@
...
+
+These services do not address anyone under the age of 13\. We do not knowingly collect personally identifiable information from children under 13\. In the case we discover that a child under 13 has provided us with personal information, we immediately delete this from our servers. If you are a parent or guardian and you are aware that your child has provided us with personal information, please contact us so that we will be able to do necessary actions.
+
+## Changes to This Privacy Policy
+
+We may update our Privacy Policy from time to time. Thus, you are advised to review this page periodically for any changes. We will notify you of any changes by posting the new Privacy Policy on this page. These changes are effective immediately after they are posted on this page.
+
+### Contact Us
+
+If you have any questions or suggestions about our Privacy Policy, do not hesitate to contact us.
```
Comment (f4ww4z) [inline privacy_policy.md]: Include a link to our Talk page and https://openmrs.org/about/contact/

File: privacy_policy.md [hunk 31]
```diff
@@ -0,0 +1,84 @@
...
+
+The terms used in this Privacy Policy are subjected to [OpenMRS License](https://openmrs.org/license/).
+
+
+## Information Collection and Use
+For a better experience, while using our service, we may require you to provide us with *personally identifiable information*, including but not limited to 
+
+1. Full name
+2. Full address
+3. Gender, birth-date and age
```
Comment (Abdelaty) [inline privacy_policy.md]: Sorry for the grammar suggestions, but it should be like this:
3. Gender, birth-date, and age

File: privacy_policy.md [hunk 32]
```diff
@@ -0,0 +1,84 @@
...
+4. Your photographs
+5. Provider information
+6. Notes on symptoms and other information
+7. Medical diagnostics and vitals (body temperature, pulse rate, blood pressure, weight, height, and respiratory rate)
+8. Medical treatment information
+9. Medical appointments
+
+The information that we request will be retained by us and used as described in this privacy policy.
+
+Link to privacy policy of third party service providers used by the app
```
Comment (Abdelaty) [inline privacy_policy.md]: "Link to the privacy policy of third party service providers used by the app"

File: privacy_policy.md [hunk 33]
```diff
@@ -0,0 +1,84 @@
...
+1.  Patient’s Identifiers are validated using [Mod10](https://en.wikipedia.org/wiki/Luhn_algorithm) algorithm. This is used while creating new Patient’s Identifiers and avoid errors.
+2.  Data is transmitted through a secure SSL connection
+3.  Strings are hashed using [SHA-512](https://en.wikipedia.org/wiki/SHA-2)
+
+## Data Retention
+We retain all the data generated by the Android app and web client. If an account is deactivated/deleted, we move the data related to that account away from our active database **but** is retained in our backup database.
+
+## Further Notes
+1.  We do not sell your data
+2.  All these personal details helps in creating a record of a certain patient. It helps the users of this app (the Providers) to maintain the track of the patient.
```
Comment (Abdelaty) [inline privacy_policy.md]: " 2.  All these personal details help in creating a record of a certain patient. It helps the users of this app (the Providers) to maintain the track of the patient. "

File: privacy_policy.md [hunk 34]
```diff
@@ -0,0 +1,84 @@
...
+3.  For users’ assistance we may require the personal details of the patients (such as date-of-birth, name, etc) to easily track the patient and find their records.
```
Comment (Abdelaty) [inline privacy_policy.md]: " 3.  For users’ assistance, we may require the personal details of the patients (such as date-of-birth, name, etc) to easily track the patient and find their records. "

File: privacy_policy.md [hunk 35]
```diff
@@ -0,0 +1,84 @@
...
+4.  We may aggregate and combine all personal information that we receive through this app (see the _Information Collection and Use_) as well as from the app’s log and usage data and use them to provide better experience across our platforms (both web and mobile).
```
Comment (Abdelaty) [inline privacy_policy.md]: "4.  We may aggregate and combine all personal information that we receive through this app (see the _Information Collection and Use_) as well as from the app’s log and usage data and use them to provide **a better** experience across our platforms (both web and mobile). "

File: privacy_policy.md [hunk 36]
```diff
@@ -0,0 +1,84 @@
...
+
+## Information Sharing and Disclosure
+1.  The Patients are not our immediate/primary/end users of this app
```
Comment (Abdelaty) [inline privacy_policy.md]: " 1.  Patients are not our immediate/primary/end users of this app "

**Comment by Kapil706** [comment]
In the pull request template The X needed to be covered clearly otherwise during the markdown compilation it will not turn into symbol that needed

File: privacy_policy.md (hunk 36 above)
Comment (f4ww4z) [reply privacy_policy.md]: @Abdelaty you can suggest changes by pressing `Ctrl + G` when commenting on a line.

**Comment by f4ww4z** [comment]
@deepak140596 Did you take a look at @Abdelaty 's comments? They are all valid points.

**Comment by deepak140596** [comment]
Thanks @Kapil706 for pointing out the error on the description.

**Comment by deepak140596** [comment]
@f4ww4z , @Abdelaty , Thanks for the feedback. I will update the commit as soon as I can.

**Pull request merged**